build-libs/redirects.js

```diff
@@ -56,6 +59,26 @@ function addHostCondition(redirects, productSlug) {
 		if (productSlug == PROXIED_PRODUCT) {
 			return redirect
 		}
+
+		// If the productSlug is NOT a beta product, it is GA, so handle the redirect appropriately
+		if (!config['dev_dot.beta_product_slugs'].includes(productSlug)) {
```

Comment:
🤔 What's enabling us to access `dev_dot.beta_product_slugs` on config?

Reply:
It's defined here: https://github.com/hashicorp/dev-portal/pull/1144/files#diff-00a4f13d760b82b998bcae3d31b5fbd8d2e9edf400c352dfa980ecb82fdf8024R18

Reply:
Ah, `flat` 🎉